src/bugreporter/odemis_bugreporter.py

```diff
@@ -318,6 +318,23 @@ def compress_files(self):
             if delphi_calib_reps and (time.time() - os.path.getmtime(delphi_calib_reps[-1])) / 3600 < 24:
                 files.append(delphi_calib_reps[-1])
 
+            # Add the latest fastem-calibration image if it's possibly related (ie, less than a week old)
+            fastem_calib_dir = os.path.join(home_dir, "development/fastem-calibrations/images/cell-translation")
+            fastem_calib_reps = glob(os.path.join(fastem_calib_dir, '*'))
+            fastem_calib_reps.sort(key=os.path.getmtime, reverse=True)
+            # If the last calibration failed, the image might not have been created, then get the last available image.
+            for f in fastem_calib_reps:
+                calib_file = os.path.join(f, "plot_cell_translations.png")
+                if os.path.isfile(calib_file) and (time.time() - os.path.getmtime(f)) / 3600 < (24 * 7):
+                    files.append(calib_file)
+                    break
+
+            # add the contents of the entire content of odemis-status dir if available,
+            # no need to filter, because the status gets overwritten.
+            odemis_status_dir = os.path.join(home_dir, "odemis-status")
+            if os.path.isdir(odemis_status_dir):
+                files.append(odemis_status_dir)
```

Comment:
@pieleric is it save to just add the entire contents of the odemis status dir? I could also filter for anything that ends in `*.txt`

Reply:
I think this is fine. Unless you expect to have large files (after compression). The ticketing system accepts up to 128Mb of attachment... but typically it's nicer if it's below ~20Mb.